### ISSUE TITLE
Make it possible to translate Pattypan with Translatewiki

### ISSUE THREAD

**Issue by Abbe98** [yarl/pattypan#160]
Currently contributing to translations require knowledge of git and Github. Many translations aren't up-to-date either.

We should make it possible to translate Pattypan using Translatewiki.

**Comment by Aktron** [comment]
might this be relevant? https://translatewiki.net/wiki/Setup_of_a_new_project

**Comment by Abbe98** [comment]
@Aktron yes indeed, and if someone picks this up and runs into problem I will happily refer them to someone working on Translatewiki!
